src/BasisTheory.net/Api/PermissionsApi.cs

```diff
@@ -219,10 +223,16 @@ public List<Permission> Get(string applicationType = default(string), int operat
         /// <exception cref="BasisTheory.net.Client.ApiException">Thrown when fails to make API call</exception>
         /// <param name="applicationType"> (optional)</param>
         /// <param name="operationIndex">Index associated with the operation.</param>
+        /// <param name="requestOptions"></param>
         /// <returns>ApiResponse of List&lt;Permission&gt;</returns>
-        public BasisTheory.net.Client.ApiResponse<List<Permission>> GetWithHttpInfo(string applicationType = default(string), int operationIndex = 0)
+        public BasisTheory.net.Client.ApiResponse<List<Permission>> GetWithHttpInfo(string applicationType = default(string), int operationIndex = 0, BasisTheory.net.Client.RequestOptions requestOptions = default(BasisTheory.net.Client.RequestOptions))
         {
-            BasisTheory.net.Client.RequestOptions localVarRequestOptions = new BasisTheory.net.Client.RequestOptions();
+            if(requestOptions == default(BasisTheory.net.Client.RequestOptions))
+            {
+                requestOptions = new BasisTheory.net.Client.RequestOptions();
+            }
```

Comment:
Allowing the client to pass a `RequestOptions` object. The previous template created the RequestOptions internally.